### DISCUSSION TITLE
EVI2 in TSA?

### DISCUSSION THREAD

**Discussion by JonasardoLundUniv** [davidfrantz/force#192] [Ideas]
Perhaps EVI2 can be added to INDEX in TSA module?

**Currently:
Spectral index**

    Perform the time series analysis using the specified band or index. Multiple indices can be processed ar once to avoid multiple reads of the same file. Only necessary bands will be input. You will be alerted if the index cannot be computed based on the requested SENSORS. The index SMA is a linear spectral mixture analysis and is dependent on the parameters specified in the SPECTRAL MIXTURE ANALYSIS section below.
    Type: Character list. Valid values: {BLUE,GREEN,RED,NIR,SWIR1,SWIR2,RE1,RE2,RE3,BNIR,NDVI,EVI,NBR,NDTI,ARVI,SAVI,SARVI,TC-BRIGHT,TC-GREEN,TC-WET,TC-DI,NDBI,NDWI,MNDWI,NDMI,NDSI,SMA,kNDVI,NDRE1,NDRE2,CIre,NDVIre1,NDVIre2,NDVIre3,NDVIre1n,NDVIre2n,NDVIre3n,MSRre,MSRren}_

**Comment by davidfrantz** [comment]
Hi Jonas,
yes, this would be a good addition. I will try to find some time to implement this. 
Cheers,
David

**Reply by davidfrantz** [reply]
@JonasardoLundUniv,

``EV2`` is now available in the develop branch!

Cheers,
David

**Reply by JonasardoLundUniv** [reply]
Great! Thanks.
